platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -465,25 +465,44 @@ static boolean handleMotionEvent(int eventSource, int eventAction, int buttonsMa
 	}
 
 	static boolean handleMouseEvent(final MotionEvent event) {
+
 		final int eventAction = event.getActionMasked();
```

Comment:
Unnecessary empty line.
```suggestion
	static boolean handleMouseEvent(final MotionEvent event) {
		final int eventAction = event.getActionMasked();
```